contracts/seed/Seed.sol

```diff
@@ -410,42 +418,42 @@ contract Seed {
 
     /**
       * @dev                      get the total seed amount bought
-      * @param _locker            Address of lock to find the total seed amount bought
+      * @param _funder            Address of funder to find the total seed amount bought
     */
-    function getSeedAmount(address _locker) public view returns(uint256) {
-        return tokenLocks[_locker].seedAmount;
+    function getSeedAmount(address _funder) public view returns(uint256) {
```

Comment:
I think we could reduce the size of the contract by removing these getters that are doing no more that dereferencing `funders`.  There is really no need for these getters.

Reply:
Okay, so you will directly get it from the state?

Reply:
yes